specs/chugsplash-manager.md

```diff
@@ -123,8 +131,9 @@ const getProxyByName = (
 * Cancelling a bundle MUST put the bundle in the `CANCELLED` state.
 * Cancelling a bundle MUST remove the active bundle.
 * If an executor has been selected within the last `executionLockTime` seconds (i.e. `block.timestamp` <= `timeClaimed + executionLockTime`):
-  * The ChugSplashManager MUST increase the `debt` owed to the current executor by `BUNDLE_EXECUTION_BOND_AMOUNT`.
+  * The ChugSplashManager MUST increase the `debt` owed to the current executor by `BUNDLE_EXECUTION_BOND_AMOUNT + executorBondAmount`.
   * The ChugSplashManager MUST increase the `totalDebt` by the `BUNDLE_EXECUTION_BOND_AMOUNT`.
+* Otherwise, the `totalDebt` must decrease by the `executorBondAmount`.
```

Comment:
Also unrelated to the new action type.

If we don't decrease the `totalDebt` here, this amount will be stuck in the ChugSplashManager.